index.js

```diff
@@ -1,23 +1,26 @@
 'use strict';
 let { useState, useEffect } = require('react');
 
-function getSize() {
-  return {
-    innerHeight: window.innerHeight,
-    innerWidth: window.innerWidth,
-    outerHeight: window.outerHeight,
-    outerWidth: window.outerWidth,
-  };
-}
-
 function useWindowSize() {
+  let isClient = typeof window === 'object';
   let [windowSize, setWindowSize] = useState(getSize());
 
+  function getSize() {
```

Comment:
There's no need to move this function inside `useWindowSize`, you can just do the `typeof window` check inside `getSize`.

Reply:
I create PR #8 with fix this.